root_orchestrator/resource-abstractor/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.8.5-slim-buster
+LABEL org.opencontainers.image.source https://github.com/oakestra/oakestra
+
+ADD requirements.txt /
+
+RUN pip install -r requirements.txt
+
+ADD . /
+
+# TRUE for verbose logging
+ENV FLASK_ENV=development
+ENV FLASK_DEBUG=TRUE
+
+ENV CLOUD_MONGO_URL=localhost
+ENV CLOUD_MONGO_PORT=10007
+
+ENV MY_PORT=10011
```

Comment:
Nitpick:
What does "MY" refer to here?
If it refers to the resource-abstractor (i.e. what is being built in this Dockerfile) wouldn't it be a bit clearer to use `RESOURCE_ABSTRACTOR_PORT` instead?

Reply:
I guess he's following the same structure as the other components. But as a general rule, I agree with you, we can use `RESOURCE_ABSTRACTOR_PORT` here and slowly change the other instances.